topics/terminal-usage/initial-setup/index.md

```diff
@@ -78,6 +78,17 @@ other than Windows 10), your copy of Windows does not include this terminal. In
 that case, we recommend that you use [MobaXterm][mobaxterm] for SSH. Download
 and install it from that site.
 
+MobaXterm installs a more minimal terminal than the other supported platforms,
+so copy and paste the following command into your terminal and then press enter
+before continuing with setup. This command will install the nano program, which
+is a simple text editor. You may need to confirm the installation by pressing y
+and then enter if prompted. You should verify that you see an [image similar to
+this][mobaxterm-startup] before running the command.
+
+{% highlight bash %}
+apt-cyg install nano
+{% endhighlight %}
+
 If that number is 14316 or greater, you should follow these instructions to
```

Comment:
This number seems out of context now

Reply:
14316 is the first build of Windows 10 that includes the built-in Linux subsystem and terminal :)

https://blogs.windows.com/windowsexperience/2016/04/06/announcing-windows-10-insider-preview-build-14316/

Reply:
Oh, I know-- I only meant it's separate too far from the time it's introduced!

Reply:
I think just changing the order of the Bash for Windows and MobaXTerm paragraphs should fix things. Then I can add the new paragraph at the end and everything stays in context.